src/improvements/tasks/MultisigTask.sol

```diff
@@ -1107,11 +1113,13 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
     /// ensuring subsequent calldata generation uses the correct
     /// nonce values even after the task has been executed locally.
     function _overrideState(string memory taskConfigFilePath) private {
-        _applyStateOverrides(taskConfigFilePath);
+        _readStateOverridesFromConfig(taskConfigFilePath); // Sets global '_stateOverrides' variable.
```

Comment:
change name of this function as it implies it's just a read op when it's actually a write op to the global state variable.